collector/monitors/test_data/slashing_success_response_blocks_jailed_tombstoned.json

```diff
@@ -0,0 +1,8 @@
+{
+  "address": "terravalcons1ezj3lps8nqwytt42at2sgt7seq9hk708g0spyk",
+  "start_height": "2521439",
+  "index_offset": "1411437",
+  "jailed_until": "2022-01-01T00:00:00Z",
```

Comment:
test TestSuccessfulRequestWithSlashing will fail in 6 months

Reply:
changed to `2032-01-01T00:00:00Z`, thanks